games-simulation/simgear/simgear-2018.2.2.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Simulator Construction Gear"
+DESCRIPTION="SimGear is a set of open-source libraries designed \
+to be used as building blocks for quickly assembling 3D simulations, \
+games, and visualization applications."
+HOMEPAGE="https://www.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/simgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="b5d516d73c64be4fed6a8155fab44b232eaa62ab4aaec5f500e2b07ad9a1e850"
+PATCHES="simgear-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	simgear$secondaryArchSuffix = $portVersion compat >= 2
+	lib:simgear$secondaryArchSuffix = $portVersion compat >= 2
+	devel:simgear$secondaryArchSuffix = $portVersion compat >= 2
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:plib$secondaryArchSuffix
+	lib:openscenegraph$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:openal$secondaryArchSuffix
+	lib:boost$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:plib$secondaryArchSuffix
+	devel:openscenegraph$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:openal$secondaryArchSuffix
+	devel:boost$secondaryArchSuffix
+	"
+

```

Comment:
We prefer to not have an empty line before `BUILD_PREREQUIRES`. But we like to have one before `BUILD_REQUIRES`, so you can keep that one.
```diff
 	devel:boost$secondaryArchSuffix
 	"
-
 BUILD_PREREQUIRES="
 	haiku${secondaryArchSuffix}_devel
```